test/js-api/jsapi.js

```diff
@@ -686,8 +686,7 @@ assertCompileError([1], TypeError);
 assertCompileError([{}], TypeError);
 assertCompileError([new Uint8Array()], CompileError);
 assertCompileError([new ArrayBuffer()], CompileError);
-assertCompileError([new Uint8Array("hi!")], CompileError);
-assertCompileError([new ArrayBuffer("hi!")], CompileError);
+assertCompileError([new Uint8Array([0xC0, 0xDE, 0xFE, 0xFE]), CompileError);
```

Comment:
Mind defining a const kInvalidWasmBuffer = [0xC0, 0xDE, 0xFE, 0xFE], and then we can just reuse this? We may have a few other places we're doing this, too.